endpoints/openrtb2/auction_test.go

```diff
@@ -5618,6 +5618,50 @@ func TestValidateOrFillCookieDeprecation(t *testing.T) {
 	}
 }
 
+func TestSetGPCImplicitly(t *testing.T) {
+	testCases := []struct {
+		description string
+		headerValue string
+		expectedGPC string
+		expectError bool
+	}{
+		{
+			description: "Sec-GPC header set to '1', GPC should be set to '1'",
+			headerValue: "1",
+			expectedGPC: "1",
+			expectError: false,
+		},
+	}
+
+	for _, test := range testCases {
+		t.Run(test.description, func(t *testing.T) {
+			httpReq := &http.Request{
+				Header: http.Header{
+					"Sec-GPC": []string{test.headerValue},
+				},
+			}
+
+			r := &openrtb_ext.RequestWrapper{
+				BidRequest: &openrtb2.BidRequest{}, // what I should add here???
```

Comment:
I would do something like this:
```
r := &openrtb_ext.RequestWrapper{
	BidRequest: &openrtb2.BidRequest{
		Regs: test.regs,
	},
}
```

You would then need to define the following field on your test case struct that defines the original state of `regs`:
`regs *openrtb2.Regs`